.github/workflows/test.yml

```diff
@@ -95,3 +95,48 @@ jobs:
       - name: Build
         run: pnpm run build
         working-directory: web
+      # - name: Install Playwright
+      #   run: pnpm run e2e-install
+      #   working-directory: web
+      # - name: Run Playwright tests
+      #   run: pnpm run e2e
+      #   working-directory: web
+      #   env:
+      #     # This might depend on your test-runner/language binding
+      #     PLAYWRIGHT_TEST_BASE_URL: djmoxvh6xpra8.cloudfront.net/
+  playwright-tests:
+    # needs: [test-frontend]
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    # if: github.event.deployment_status.state == 'success'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 7.1.5
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+          # cache: 'pnpm'
+      # Should be installed from dependent test-frontend
+      # - name: Install dependencies
+      #   run: pnpm ci
+      #   working-directory: web
+      - name: Install Deps
+        run: pnpm i
+        working-directory: web
+      - name: Install Playwright
+        run: pnpm run e2e-install
+        working-directory: web
+      - name: Run Playwright tests
+        run: pnpm run e2e
+        working-directory: web
+        env:
+          PLAYWRIGHT_TEST_BASE_URL: djmoxvh6xpra8.cloudfront.net/ # @TODO find a way to pass this between deployment
+          TESTING_DOMAIN: '${{ secrets.TESTING_DOMAIN}}' # @TODO find a way to pass this between deployment
```

Comment:
`secrets.TESTING_DOMAIN` is okay for now - to get the benefit of testing in GitHub Actions we'll need to update this to point to the branch deployment URL (e.g. https://jordi-cf-31-add-github-action-fo.test.granted.run/). The SSL certs are currently broken here though so we need to fix those first!